src/templates/finops-hub/modules/scripts/IngestionSetup_v1_2.kql

```diff
@@ -1509,6 +1509,12 @@ Recommendations_transform_v1_2()
             'SkuSize', coalesce(SKU, SkuName),
             'SkuTerm', isoMonths(Term)
         ),
+        x_SourceType startswith 'FinOpsHubRecommendations-', bag_pack(
```

Comment:
@helderpinto I removed RecommendationDescription from this object because you also had it as a primary property. Did you have a preference on whether it should be in the property bag or outside of it? I assumed outside but let me know.

Reply:
Yes, it was details. Sorry, I mixed them up 😵‍💫

For the sake of passthru ingestion, I don't care about the properties in the details object. That said, we might want to keep this a key-value pair for simplicity. Nested objects can be harder to work with. Either way, we can figure that out later, assuming we keep this passthru.